### DISCUSSION TITLE
Hopper Disassembler

### DISCUSSION THREAD

**Discussion by jacobcxdev** [Homebrew/discussions#5080] [Casks]
Hopper Disassembler was previously removed (https://github.com/Homebrew/homebrew-cask/pull/49245), as the demo application was different to that of the full version. This no longer appears to be the case; the demo version can be 'activated' via a license file, turning it into the full version.

Therefore, I believe it would be useful for someone to add the Hopper Disassembler formula back to this repository.

### Relevant casks

hopper-disassembler

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
